microsoft-edge/progressive-web-apps-chromium/how-to/digital-goods-api.md

```diff
@@ -35,17 +35,47 @@ The Payment Request API⁠⁠ handles the actual payment transaction when a purc
 See [Payment Request API](https://developer.mozilla.org/docs/Web/API/Payment_Request_API) at MDN.
 
 
+<!-- ====================================================================== -->
+## Enable the Digital Goods API
+
+The Digital Goods API is currently available in pre-release versions of Microsoft Edge.  To use the API, either enable the feature locally in Microsoft Edge, or register in the Digital Goods API origin trial to test the API in production.
+
+
+<!-- ------------------------------ -->
+#### Enable the feature locally
+
+To enable the Digital Goods API on your developer machine only, for testing:
+
+TODO: How do you enable the flags `msWebAppDigitalGoods`, `AppStoreBilling` and `DigitalGoodsApi`?  They are excluded from `edge://flags`.
```

Comment:
How do you enable the flags `msWebAppDigitalGoods`, `AppStoreBilling` and `DigitalGoodsApi`? They are excluded from `edge://flags`.
@runyuany 
Seems like these are required - what do we need to tell users at this time, to make the API available - say use a specific prerelease of Edge?
@captainbrosset

Reply:
The three flags are currently enabled as part of a running experiment and will be enabled by default in the code once we reach the shipping stage. Developers can start using the API at this stage by running `Dev` or `Canary` Edge versions `115.0.1860.0` or later. Alternatively, the API can be used in the Beta and Stable version of Edge by launching it with the following command-line argument: `--enable-features=msWebAppDigitalGoods, AppStoreBilling, DigitalGoodsApi`.

Yes, the three of flags are all required, and I would tell users to use Dev/Canary Edge at this time, since launching Edge with cmd line arguments is not a common way for most of the users.

Reply:
```
To enable the Digital Goods API on your developer machine only, for testing:

The `msWebAppDigitalGoods`, `AppStoreBilling` and `DigitalGoodsApi` flags are currently enabled as part of a running experiment, and will be enabled by default in the code once this feature reaches Edge Stable release.  You can start using the API now by running Edge Dev or Edge Canary version 115.0.1860.0 or later.

Alternatively, you can use the API in your own Beta and Stable version of Microsoft Edge by launching Edge with the following command-line argument:

`--enable-features=msWebAppDigitalGoods, AppStoreBilling, DigitalGoodsApi`
```